detector.js

```diff
@@ -265,7 +269,8 @@ export default class Detector {
             if (faceBox.xMin < handBox.xMin && handBox.xMax < faceBox.xMax) {
                 // The hand bounding box is with in the face box,
                 // which means the hand is in front of the face
-                detected = minDistance.d < 10;
+                // detected = minDistance.d < 10;
+                detected = minDistance.d < 25;
```

Comment:
I merged wenbo's changes, I think we will probably need retune these value a little bit.